homeassistant/components/cast/media_player.py

```diff
@@ -184,13 +184,15 @@ def __init__(self, cast_info: ChromecastInfo) -> None:
         self._cast_view_remove_handler = None
         self._attr_unique_id = str(cast_info.uuid)
         self._attr_name = cast_info.friendly_name
-        if cast_info.cast_info.model_name != "Google Cast Group":
-            self._attr_device_info = DeviceInfo(
-                identifiers={(CAST_DOMAIN, str(cast_info.uuid).replace("-", ""))},
-                manufacturer=str(cast_info.cast_info.manufacturer),
-                model=cast_info.cast_info.model_name,
-                name=str(cast_info.friendly_name),
-            )
+        self._attr_device_info = DeviceInfo(
+            entry_type=dr.DeviceEntryType.SERVICE
+            if cast_info.cast_info.cast_type == pychromecast.const.CAST_TYPE_GROUP
+            else None,
```

Comment:
This would be more readable if it was not a multiline ternary.

Reply:
Right, removed the ternary.